.appveyor.yml

```diff
@@ -37,10 +58,15 @@ build_script:
 after_build:
 - ps:  fsutil behavior set disablelastaccess 1 # Disable Access time feature on Windows (better performance)
 - ps:  clcache -z
+#- 7z a bitcoin-%APPVEYOR_BUILD_VERSION%.zip %APPVEYOR_BUILD_FOLDER%\build_msvc\%platform%\%configuration%\*.exe
+before_test:
+- cmd:  move /Y "build_msvc\%PLATFORM%\%CONFIGURATION%\*.exe" src
```

Comment:
Why is this needed?

Reply:
Please see:

*  [MSVC] Copy build output to src/ automatically after build #16308

Reply:
Good spot. No manual step required to move the binaries for testing since Nicolas added the equivalent task to the project file.